factory/mock/builtInFunctionFactoryMock.go

```diff
@@ -0,0 +1,70 @@
+package mock
```

Comment:
1. Please move this in testscommon
2. Why define type BuiltInFunctionFactory interface here? It is not being used in project anyway

Reply:
1. moved to testcommon
2. Remove. Initially wanted to have just 3 methods from the Factory, but changed it.